src/nth.cpp

```diff
@@ -0,0 +1,15 @@
+#include "nth.hpp"
+
+std::string_view impl::suffix_for(size_t value) {
+    if (value > 4 && value < 20)
+        return "th";
+
+    if (value % 10 == 1)
+        return "st";
+    if (value % 10 == 2)
+        return "nd";
+    if (value % 10 == 3)
```

Comment:
I think this may be wrong for numbers like 113, which should be th, but it'll come out as 113rd. It probably needs to go into the check on line 4.

Reply:
This is code from the original (or near enough). But good point, let me write and fix a failing test!